extensions/3rdparty/DynamicFunctions/DynamicFunctions.php

```diff
@@ -66,7 +66,7 @@ class ExtDynamicFunctions
 	 * @param Parser $parser
 	 * @return bool
 	 */
-    function arg( &$parser, $name = '', $default = '' )
+    function arg( $parser, $name = '', $default = '' )
     {
         global $wgRequest;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)